resources/jolokia-access.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
When I read this file name I thought it had something to do with an access log, since we call the request logs for puppetserver `puppetserver-access.log`, `file-sync-access.log`, etc.

Would it be possible to rename this as `jolokia-access-policy.xml` or something like that, or is `jolokia-access.xml` pretty much always the file used and so we don't want to change that?